### DISCUSSION TITLE
Model file upload error

### DISCUSSION THREAD

**Discussion by Seohare** [vladmandic/sdnext#3109] [Q&A]
I put the model file in \automatic\models\Stable-diffusion and ran it and it gets an error as shown in the picture below. How do I fix it?

![error](https://github.com/vladmandic/automatic/assets/99063049/b36e8faf-c523-4d1e-84b1-ffece4252b99)

**Comment by Disty0** [comment]
Did you manually set a diffusers pipeline? Set it back to auto if you did.

**Reply by Seohare** [reply]
What is it and how do I set it up automatically?

**Comment by Disty0** [comment]
Seems like this model has broken keys in it. Use Models -> Convert tab to fix it's broken keys. (Enable fix clip too.)

**Reply by Seohare** [reply]
It's the same

**Reply by Disty0** [reply]
Did you load the fixed model?

**Reply by Seohare** [reply]
What is it?

**Reply by Disty0** [reply]
Convert model tab will create a new model. Use that one.

**Comment by vladmandic** [comment]
also, never screenshot logs - logs are text - we need them to be text so we can actually look at them.
and post longer log leading to error, not just final error message.
